### ISSUE TITLE
can i train Yolo with an imageset larger than 1920x1080 

### ISSUE THREAD

**Issue by mkhegazy** [pjreddie/darknet#368]
Hello, i am fairly new to yolo i am trying to use it and i wanted to ask a question. i am in the process of bounding boxes using YOLO_Mark. i have a set of images (around 200 image) i have are taken from my phone so they are usually around 4032x3024 . so the question is, should i convert these images to lower resolution (1000x1000 or even lower) or i can use it normally 4032x3024 and wont be an issue

**Comment by baristahell** [comment]
There are several issues : 
First, you have to compute your proposed anchors to fit the larger final grid and input resolution so all of your objetcs are still visible by the network. Then, will you have enough memory to store the corresponding net weights and activations while training and testing?

**Comment by mkhegazy** [comment]
ok. since i am having the issue with the memory size in the first place then i have no option rather than decreasing the image dimensions. stick to 800x800 ?

**Comment by baristahell** [comment]
Be careful when you resize, to not crush your small objects. I'm using 1920*1080 images that are resized to 832*416 to get a 26*13 grid, i guess 832*832 would be good for you.

**Comment by mkhegazy** [comment]
ok great thank you very much . i started working on the resize. (around 200 images for a start :D) when i am done i will let you know how did it go . thank you again

**Comment by lqian** [comment]
@baristahell A interesting quest about grid number,  as you mention, 832x416 get 26x13 grid. The YOLO net configs 416x416 as input. Its config file does not have any parameter about the grid number. Does the YOLO net computes the grid number automatically?

**Comment by baristahell** [comment]
Yes, the grid number is "set" according to your input size. That is because of the successive conv/pooling, reducing the feature map size to a ratio of 1/32th of the input.

**Comment by mkhegazy** [comment]
i used the Application called Yolo_mark. interestingly it changed the size from 4094x3024 to 1000x800

**Comment by maksimkolesov** [comment]
@baristahell Can u give more info about compute anchor boxes? Have some troubles with understanding this:(

**Comment by kmsravindra** [comment]
@baristahell, about this comment "Be careful when you resize, to not crush your small objects. I'm using 1920x1080 images that are resized to 832x416 to get a 26x13 grid, i guess 832x832 (so a 26x26 grid) would be good for you."

When you resize from 1920 x 1080 to 832 x 416, the aspect ratio is not the same? Did you use the network to predict on 832 x 416 itself? OR on 1920 x 1080?

**Comment by matanhs** [comment]
Hi, Im also trying to increase the image size for tiny yolo-v2 on pascal dataset. I initialized parameters from the 416x416 checkpoint and recomputed the anchors appropriately (new input size is arbitrarily set to 960x960). However, the quality of the model seems to be underwhelming. Anyone cares to share your large image model accuracy?

**Comment by jorgegaticav** [comment]
> There are several issues :
> First, you have to compute your proposed anchors to fit the larger final grid and input resolution so all of your objetcs are still visible by the network. Then, will you have enough memory to store the corresponding net weights and activations while training and testing?
> I'd say, if you can find a way to cut them in 4 without impeding your goal (boxes cut in half and annoyances like that), definitely go for it.

Do you have more info about how to compute the anchors? I'm using 2048x2048 res images.
